rlberry/agents/agent.py

```diff
@@ -276,49 +291,13 @@ def load(cls, filename, **kwargs):
 
 
 class AgentWithSimplePolicy(Agent):
-    """A subclass of Agent meant to help design simple agents
-
-    Implement a policy() method, and a simple evaluation
-    method (Monte-Carlo policy evaluation).
-
-    The policy() method takes an observation as input and returns an action.
+    """Interface for agents whose policy is a function of observations only.
 
+    Requires a :meth:`policy` method, and a simple evaluation method (Monte-Carlo policy evaluation).
 
-    Parameters
-    ----------
-    env : gym.Env or tuple (constructor, kwargs)
-        Environment used to fit the agent.
-    eval_env : gym.Env or tuple (constructor, kwargs)
-        Environment on which to evaluate the agent. If None, copied from env.
-    copy_env : bool
-        If true, makes a deep copy of the environment.
-    seeder : rlberry.seeding.Seeder, int, or None
-        Object for random number generation.
-    _execution_metadata : ExecutionMetadata (optional)
-        Extra information about agent execution (e.g. about which is the process id where the agent is running).
-    _default_writer_kwargs : dict (optional)
-        Parameters to initialize DefaultWriter (attribute self.writer).
-    .. note::
-        Classes that implement this interface should send ``**kwargs`` to :code:`Agent.__init__()`
-
```

Comment:
I agree that this is a copy/paste of Agent, hence redundant, but I would still keep it because people don't necessarily think of going to the Agent doc to have a doc on AgentWithSimplePolicy. Moreover, if you want this docstring to be usable in python with `?AgentWithSimplePolicy`, it would be annoying to have to import Agent to do `?Agent` to know the doc on AgentWithSimplePolicy. The best solution would be to automatically inherit in the doc but I don't know if this is doable. probably not.